app/service-worker.js

```diff
@@ -1,4 +1,21 @@
-// This file is intentionally without code.
-// It's present so that service worker registration will work when serving from the 'app' directory.
-// The version of service-worker.js that's present in the 'dist' directory is automatically
-// generated by the 'generate-service-worker' gulp task, and contains code to precache resources.
+'use strict';
+
+/* eslint-env serviceworker */
+/* global WorkboxSW */
+
+// This script is added to app/third_party/ by gulp and swapped for a prod
+// file name in production build.
+importScripts('/third_party/workbox-sw/workbox-sw.dev.js');
+
+self.workbox = self.workbox || {};
+self.workbox.LOG_LEVEL = 0;
+
+const workbox = new WorkboxSW();
+
+// workbox-build will swap out the empty array with a list of files to precache.
+workbox.precache([]);
+
+// Register route for Google static files.
+workbox.router.registerRoute(
+  /.*(?:googleapis|gstatic)\.com.*$/,
```

Comment:
You don't need the trailing `.*$` characters, since a partial match is fine (as long as it begins with the first character for cross-origin requests).